src/main/java/cz/fi/muni/pa165/esports/entity/Match.java

```diff
@@ -1,4 +1,124 @@
-package cz.fi.muni.pa165.esports.entity;
-
-public class Match {
-}
+package cz.fi.muni.pa165.entity;
+
+import java.util.*;
+
+import javax.persistence.*;
+//import javax.validation.constraints.NotNull;
+
+/*
+    @Author Elena Álvarez
+*/
+
+@Entity
+public class Match {
+    @Id
+    @GeneratedValue(strategy=GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;

```

Comment:
shouldn't this have @unique? asking because the dao.findByName() return only one result